components/calendar/src/ixdtf.rs

```diff
@@ -12,7 +12,7 @@ use ixdtf::ParserError;
 /// An error returned from parsing an IXDTF string to an `icu_calendar` type.
 #[derive(Debug)]
 #[non_exhaustive]
-pub enum FromStrError {
+pub enum ParseError {
     /// Syntax error in the IXDTF string.
     Syntax(ParserError),
```

Comment:
follow-up: this should also be `ParseError` instead of `ParserError`.

Reply:
https://github.com/unicode-org/icu4x/issues/2127#issuecomment-2240155081